tests/ui/manual_is_ascii_check.rs

```diff
@@ -15,6 +15,7 @@ fn main() {
     assert!(matches!('x', 'A'..='Z' | 'a'..='z'));
 
     assert!(matches!('x', 'A'..='Z' | 'a'..='z' | '_'));
+    assert!((b'0'..=b'9').contains(&b'0'));
```

Comment:
Could you add more tests with non-exclusive ranges like this: `(b'0'..b'9').contains(&b'0')`.

It would also be good to test all the other ranges:
`'A'..='Z'`, `'a'..='z'` ...